package com.ecom.repository.order;

import com.ecom.models.order.Order;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

import java.util.List;
import java.util.Optional;

@Repository
public interface OrderRepository extends MongoRepository<Order , String> {
    List<Order> findByUserId(String userId);

    Optional<Order> findByMoyasarPaymentId(String moyasarPaymentId);
}
